deploy/env-development.json

```diff
@@ -117,6 +135,103 @@
                 "[resourceId('Microsoft.Network/virtualNetworks', 'dev-vnet')]"
             ]
         },
+        {
+            "sku": {
+                "name": "Standard_D2s_v3",
+                "tier": "Standard",
+                "capacity": 1
+            },
+            "properties": {
+                "upgradePolicy": {
+                    "mode": "Manual"
+                },
+                "virtualMachineProfile": {
+                    "osProfile": {
+                        "computerNamePrefix": "ci-",
+                        "adminUsername": "cloud-user",
+                        "linuxConfiguration": {
+                            "disablePasswordAuthentication": true,
+                            "ssh": {
+                                "publicKeys": [
+                                    {
+                                        "path": "/home/cloud-user/.ssh/authorized_keys",
+                                        "keyData": "[parameters('sshPublicKey')]"
+                                    }
+                                ]
+                            }
+                        }
+                    },
+                    "storageProfile": {
+                        "imageReference": {
+                            "publisher": "RedHat",
+                            "offer": "RHEL",
+                            "sku": "8.1",
+                            "version": "8.1.2020020415"
```

Comment:
I'm uncomfortable that we're using RHEL 8 here and 7 elsewhere, it would be better if we could standardise...

Reply:
The issue is with `libseccomp libseccomp-devel` pkg state in rhel7. How much of priority it is as it might take ~1 day for a fix.

Reply:
Done